<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMain</class>
 <widget class="QWidget" name="WidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente TCP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Servidor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditServer"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnConectarChat">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="btnSelectFile">
     <property name="text">
      <string>Seleccionar Archivo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSendFile">
     <property name="text">
      <string>labelSendFile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSendFile">
     <property name="text">
      <string>EnviarArchivo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
